IS/publiccode.yml

```diff
@@ -0,0 +1,109 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+applicationSuite: WSO2
+categories:
+  - it-security
+  - it-service-management
+  - identity-management
+description:
+  it:
```

Comment:
As this is in English the country code should be "en" check -> https://docs.italia.it/italia/developers-italia/publiccodeyml-en/en/master/schema.core.html and search for "description"

Reply:
Updated accordingly